bbridge_client.py

```diff
@@ -48,15 +48,19 @@ def __init__(self, token, host_url="http://bbridgeapi.cloudapp.net/v1"):
         self.__sentiment_analysis_url = "{}/nlp/sentiment".format(host_url)
         self.__name_entity_recognition_url = "{}/nlp/ner".format(host_url)
 
-    def response(self, request_id, inner_type=None):
+    def response(self, request_id, return_type=None):
         """
         :type request_id: str
-        :type inner_type: type | None
+        :type return_type: type | None
         :rtype: entity.response.Response
         """
-        response = requests.get(self.__response_url, params={"id": request_id}, headers=self.__auth_header)
+        response = requests.get(self.__response_url, params={"id": request_id}, headers=self.__headers)
         if response.status_code == 200:
-            return Response(APIResponse.from_json_str(response.text, inner_type), response.status_code)
+            content = json.loads(response.text)
+            if return_type is not None:
+                type_from_json = getattr(return_type, "from_json")
+                content = type_from_json(content)
+            return Response(content, response.status_code)
```

Comment:
You can pass not `BBridgeEntity` object to `Response` but this contradicts the contract of `Response.__init__`